friendly_traceback/base_formatters.py

```diff
@@ -31,14 +31,27 @@
   to print the information in a traditional console.
 """
 import sys
-from typing import List
+from typing import Dict, List, Set
 
 from .ft_gettext import current_lang
 from . import debug_helper
 
 
 if sys.version_info >= (3, 8):
-    from typing import Protocol, TypedDict
+    from typing import Literal, Protocol, TypedDict
+
+    InfoKind = Literal[
```

Comment:
@aroberge I'm not happy with the name `InfoKind` that describes possible `include` inputs. Maybe you have a better naming suggestion?

Reply:
I will think about it.

Reply:
I've changed it to `InclusionChoice` and committed to the repository.